mono/metadata/console-unix.c

```diff
@@ -467,25 +466,25 @@ ves_icall_System_ConsoleDriver_TtySetup (MonoString *keypad, MonoString *teardow
 	if (tcsetattr (STDIN_FILENO, TCSANOW, &mono_attr) == -1)
 		return FALSE;
 
-	set_control_chars (*control_chars, mono_attr.c_cc);
+	uint32_t h;
+	set_control_chars (MONO_ARRAY_HANDLE_PIN (control_chars_arr, gchar, 0, &h), mono_attr.c_cc);
```

Comment:
probably not much difference either way, but why not pass a `MonoArrayHandle` to `set_control_chars` instead of pinning?

Reply:
I kinda think in raw pointers. I kinda think pinning is an alternate solution to handles, "as long as it is quick". We should probably move this to managed w/ fixed, but avoid pound-if in some fashion -- I really wish we had identical IL across platforms. There'd still be p/invoke or icall into runtime, but only with fixed pointers.